src/tox/config/__init__.py

```diff
@@ -747,8 +747,13 @@ def passenv(testenv_config, value):
         "eventual passenv setting.",
     )
 
+    # FIXME remove whitelist in favor of allowlist_external
```

Comment:
(Missing `s` at the end)

Reply:
We don't use fixmes in this project but instead prefer the issue tracker, please remove this. We could not remove it either way before the next major version as that would be breaking change.